ROMFS/px4fmu_common/init.d/rc.logging

```diff
@@ -4,13 +4,10 @@
 # NOTE: Script variables are declared/initialized/unset in the rcS script.
 #
 
-if ! ver hwcmp PX4_SITL
+if param greater UAVCAN_ENABLE 1
 then
-	if param greater UAVCAN_ENABLE 1
-	then
-		# Reduce logger buffer to free up some RAM for UAVCAN servers.
-		set LOGGER_BUF 6
-	fi
+	# Reduce logger buffer to free up some RAM for UAVCAN servers.
+	set LOGGER_BUF 6
```

Comment:
This isn't great for v5 and newer systems. Let's see if it can be dropped entirely once CONFIG_NFILE_DESCRIPTORS is decreased.